app/models/inventory_item.rb

```diff
@@ -1,2 +1,9 @@
 class InventoryItem < ActiveRecord::Base
+  using(:grossman)
+  after_initialize :init
```

Comment:
So just to be clear, the "goal" of this issue would be to create a UUID 's in gman_services, reference them as a one to many relationship (Foreign key).